WcaOnRails/app/models/competition.rb

```diff
@@ -29,6 +29,8 @@ class Competition < ActiveRecord::Base
            allow_nil: true,
            with_model_currency: :currency_code
 
+  scope :present, -> { where("CAST(CONCAT(endYear,'-',endMonth,'-',endDay) as Datetime) >= ?", Date.today) }
```

Comment:
You could also make an `ended` scope and use `not.ended` in the code.

(I know this isn't important. Sometimes you gotta bikeshed.)